CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2021-03-22
+
+This release is rolling out to all regions - ETA for conclusion 2021-03-31 for public cloud.
+
+### Announcements
+
+* Once GA AKS will default to its new [GPU specialized image](https://aka.ms/aks/specialized-gpu-image) as the supported option for GPU-capable agent nodes.
+* Kubernetes version 1.17 will be deprecated in the last week of March 2021.
+* Before k8s 1.20 a bug would allow exec probes to run indefinitely, ignoring any timeoutSeconds configuration value. The previous buggy behavior has been fixed, and timeouts are now enforced. Additionally, this change introduces a new default timeout of 1 second. Please audit all your existing exec probes to make sure that it is appropriate to enforce a 1 second timeout. If not, please provide an explicit timeoutSeconds value that is appropriate for each [exec probe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes).
+* CSI Drivers will become default for [Kubernetes versions 1.21+](https://docs.microsoft.com/azure/aks/csi-storage-drivers).
+* Previous [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) deprecation was June 30th 2021. To better align with Kubernetes Upstream pod security policy (preview) deprecation will begin with Kubernetes version 1.21, with its removal in version 1.25. As Kubernetes Upstream approaches that milestone, the Kubernetes community will be working to document viable alternatives.
+* For all AKS clusters using kubernetes v1.20+, coredns will be upgraded to version 1.8.3. This will remove resyncperiodand upstream from the Kubernetes plugin.
+
+### Release Notes
+
+* Bug Fixes
+  * Fixed an issue regarding indecisiveness in Kubernetes versions and the auto-upgrade feature in ARM templates. Read more [here](https://github.com/Azure/AKS/issues/2138).
```

Comment:
a) Good job capturing the issue.
b) Make sure we update them when included (did this now for this one, check it out)

Reply:
Noted. Will remember this for next time.